grammar.md

```diff
@@ -23,7 +23,7 @@ char               = (* Any UTF-8 code-point *)
 char_ascii         = 'a'...'z'
                    | 'A'...'Z'
 
-identifier         = char_ascii [ { char_ascii | digit | '_' | '!' | '?' } ]
+identifier         = char_ascii [ char_ascii | digit | '_' | '!' | '?' ]
```

Comment:
This is in fact wrong.
The `[]` brackets are for optional rules, the `{}` brackets are for repeated rules.
If you remove the `{}` brackets, the rule only allows for two characters max.